en/docs/Learn/APIGateway/MessageMediation/change-the-default-mediation-flow-of-api-requests.md

```diff
@@ -59,329 +59,192 @@ Let's see how to create a custom sequence using the WSO2 API Manager Tooling Pl
-                      "low": "46",
-                      "text": "Partly Cloudy"
-                    },
-                    {
-                      "code": "30",
-                      "date": "09 May 2017",
-                      "day": "Tue",
-                      "high": "69",
-                      "low": "48",
-                      "text": "Partly Cloudy"
-                    },
-                    {
-                      "code": "28",
-                      "date": "10 May 2017",
-                      "day": "Wed",
-                      "high": "70",
-                      "low": "52",
-                      "text": "Mostly Cloudy"
-                    },
-                    {
-                      "code": "30",
-                      "date": "11 May 2017",
-                      "day": "Thu",
-                      "high": "72",
-                      "low": "52",
-                      "text": "Partly Cloudy"
-                    },
-                    {
-                      "code": "30",
-                      "date": "12 May 2017",
-                      "day": "Fri",
-                      "high": "72",
-                      "low": "48",
-                      "text": "Partly Cloudy"
-                    },
-                    {
-                      "code": "34",
-                      "date": "13 May 2017",
-                      "day": "Sat",
-                      "high": "71",
-                      "low": "46",
-                      "text": "Mostly Sunny"
-                    }
-                  ],
-                  "description": "<![CDATA[<img src=\"http://l.yimg.com/a/i/us/we/52/33.gif\"/>\n<BR />\n<b>Current Conditions:</b>\n<BR />Mostly Clear\n<BR />\n<BR />\n<b>Forecast:</b>\n<BR /> Thu - Partly Cloudy. High: 74Low: 55\n<BR /> Fri - Mostly Cloudy. High: 71Low: 53\n<BR /> Sat - Partly Cloudy. High: 65Low: 47\n<BR /> Sun - Rain. High: 62Low: 48\n<BR /> Mon - Partly Cloudy. High: 69Low: 46\n<BR />\n<BR />\n<a href=\"http://us.rd.yahoo.com/dailynews/rss/weather/Country__Country/*https://weather.yahoo.com/country/state/city-12797499/\">Full Forecast at Yahoo! Weather</a>\n<BR />\n<BR />\n(provided by <a href=\"http://www.weather.com\" >The Weather Channel</a>)\n<BR />\n]]>",
-                  "guid": {
-                    "isPermaLink": "false"
-                  }
-                }
-              }
-            }
-          }
-        }
-    ```
+    ![](../../../assets/img/Learn/pizzashack-logger.png)
 
 In this tutorial, you created a sequence to change the default mediation flow of API requests, deployed it in the API Gateway and invoked an API using the custom mediation flow.
 
 !!! note
-Please note that following mediators are not usable within custom sequences since they are not supported by API Gateway custom medications.
+    Please note that following mediators are not usable within custom sequences since they are not supported by API Gateway custom medications.
```

Comment:
add this as a important admonition.
The following mediators **can not** be used within custom sequences.
    -   Call mediator in non-blocking mode
    -   Send mediator